bin/ingest-genbank

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -euo pipefail
+
+: "${S3_SRC:=s3://nextstrain-data}"
```

Comment:
I think this wants to include an object prefix so things are a bit more clear. I'd suggest `s3://nextstrain-data/ncov` which then with the full paths below makes objects like `s3://nextstrain-data/ncov/genbank.ndjson.gz`.